github/libp2p.yml

```diff
@@ -2067,6 +2067,8 @@ repositories:
     delete_branch_on_merge: false
     description: libp2p implementation in Go
     files:
+      .github/workflows/semantic-pull-request.yml:
+        content: .github/workflows/semantic-pull-request.yml
```

Comment:
```suggestion
```

Reply:
Why not? It would be nice to follow this so we can auto-highlight breaking changes.

Reply:
It seems like a major annoyance (especially the PR title character limit).
We had similar discussions when we considered which linters to use in Unified CI. Yes, more linters can be helpful, but if you enable too many of them, they'll get in the way of productive work. That's why we only have `go vet` and `staticcheck`.

Reply:
> Why not? It would be nice to follow this so we can auto-highlight breaking changes.

FWIW, we are unlikely to use the "highlight" breaking changes bit in any form of automated tooling because a PR can easily touch multiple modules and it is thus not clear, which one this refers to once the PR is squashed.

That being said, it is still nice to highlight.

Reply:
For example, this one would fail: https://github.com/libp2p/go-libp2p-backoff/pull/11

Reply:
Yep, we have had an occasional dependabot PR with too long titles. In those cases, I've just shortened it.

Reply:
In the go-libp2p meeting today, we decided to not enable this in go-libp2p for now.